packages/sync/src/class-replicastore.php

```diff
@@ -518,23 +520,8 @@ public function comments_checksum( $min_id = null, $max_id = null ) {
 	 * @return int The checksum.
 	 */
 	public function comment_meta_checksum( $min_id = null, $max_id = null ) {
-		global $wpdb;
-		return $this->table_checksum( $wpdb->commentmeta, Defaults::$default_comment_meta_checksum_columns, 'meta_id', Settings::get_whitelisted_comment_meta_sql(), $min_id, $max_id );
-	}
-
-	/**
-	 * Retrieve the checksum for all options.
-	 *
-	 * @access public
-	 *
-	 * @return int The checksum.
-	 */
-	public function options_checksum() {
```

Comment:
Is there a reason we are removing the options logic that previously existed?

Reply:
I didn't see it used on the WPCOM side, so added it as a code cleanup. We can always restore it if we need it.